### PR TITLE
WSO2 theme and font upgrade

### PR DESCRIPTION
**Pull request by erangatl**

### CODE REVIEW

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+                'lip' +
+                '>',
+            language: {
+                searchPlaceholder: 'Filter by ...',
+                search: ''
+            }
+        });
+        
+        var wso2Extend = function(settings, opts) {
+
```
Comment (ramindu90) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: @erangatl shall we remove the extra spaces

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+                        }
+                    });
+                });
+
+                //list table list/grid view toggle function
+                var toggleButton = $('[data-click-event=toggle-list-view]');
+                toggleButton.click(function () {
+                    if ($(this).attr('data-view') == 'grid') {
+                        $(this).closest('.dataTables_wrapper').find('.dataTable').addClass('grid-view');
+                        //$(this).closest('li').hide();
```
Comment (ramindu90) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: @erangatl shall we remove the commented lines ?

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 3]
```diff
@@ -0,0 +1,327 @@
+/*
+ ~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: wrong license. ( license year etc missing)

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+            settings.wso2 = this;
+            this._constructor();
+
+        };
+        
+        wso2Extend.prototype = {
+            
+            _constructor: function(){
+                
+		        var dt = this.s.dt;
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: wrong formatting

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+                var elem = $('table', dt.table().container());
+
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: remove unnecessary new lines

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+                $('.sort-row th', elem).each(function () {
+                    if (!$(this).hasClass('no-sort')) {
+                        dropdownmenu.append('<li><a href="#' + $(this).html() + '" data-column="' + $(this).index() + '">' + $(this).html() + '</a></li>');
+                    }
+                });
+
+                //Append advance operations to list table toolbar
+                $('.dataTable.list-table').closest('.dataTables_wrapper').find('.dataTablesTop .dataTables_toolbar').html('' +
+                    '<ul class="nav nav-pills navbar-right remove-margin" role="tablist">' +
+                    '<li><button data-click-event="toggle-select" class="btn btn-default btn-primary">Select</li>' +
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Please check whether these exceed the 120 limit

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 7]
```diff
@@ -0,0 +1,327 @@
...
+                //list table list/grid view toggle function
+                var toggleButton = $('[data-click-event=toggle-list-view]');
+                toggleButton.click(function () {
+                    if ($(this).attr('data-view') == 'grid') {
+                        $(this).closest('.dataTables_wrapper').find('.dataTable').addClass('grid-view');
+                        //$(this).closest('li').hide();
+                        //$(this).closest('li').siblings().show();
+                    } else {
+                        $(this).closest('.dataTables_wrapper').find('.dataTable').removeClass('grid-view');
+                        //$(this).closest('li').hide();
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: remove commented lines

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 8]
```diff
@@ -0,0 +1,327 @@
...
+        $.fn.DataTable.wso2Extend = wso2Extend;
+
+        // Attach a listener to the document which listens for DataTables initialisation
+        // events so we can automatically initialise
+        $(document).on('preInit.dt.dte', function(e, settings, json) {            
+            if (e.namespace !== 'dt') {
+                return;
+            }
+
+//            if ( $(settings.nTable).hasClass('dte') ||
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: remove commented lines

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 9]
```diff
@@ -0,0 +1,327 @@
...
+//                 DataTable.defaults.wso2 ||
+//            ) {
+                var init = settings.oInit.wso2;
+
+                if (init !== false) {
+                    new wso2Extend (settings, $.isPlainObject(init) ? init : {});
+                }
+            //}
+        });
+
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: remove unnecessary new lines. Please fix elsewhere.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js [hunk 10]
```diff
@@ -0,0 +1,327 @@
...
+    else if (typeof exports === 'object'){
+        // Node/CommonJS
+        factory(require('jquery'), require('datatables'));
+    }
+    else if (jQuery && !jQuery.fn.dataTable.wso2Extend) {
+        // Otherwise simply initialise as normal, stopping multiple evaluation
+        factory(jQuery, jQuery.fn.dataTable);
+    }
+
+})(window, document);
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: new line required.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/noty/noty-wso2.js [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+    },
+    callback: {
+        onShow : function() {
+
+        },
+        onClose: function() {
+
+        }
+    }
+};
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/noty/noty-wso2.js]: new line required. Please fix elsewhere.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/images/loading.svg [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+     viewBox="0 0 14 14" enable-background="new 0 0 14 14" xml:space="preserve">
+    <path class="circle" stroke-width="1.4" stroke-miterlimit="10" d="M6.534,0.748C7.546,0.683,8.578,0.836,9.508,1.25 c1.903,0.807,3.339,2.615,3.685,4.654c0.244,1.363,0.028,2.807-0.624,4.031c-0.851,1.635-2.458,2.852-4.266,3.222 c-1.189,0.25-2.45,0.152-3.583-0.289c-1.095-0.423-2.066-1.16-2.765-2.101C1.213,9.78,0.774,8.568,0.718,7.335 C0.634,5.866,1.094,4.372,1.993,3.207C3.064,1.788,4.76,0.867,6.534,0.748z"/>
+    <path class="pulse-line" stroke-width="0.55" stroke-miterlimit="10" d="M12.602,7.006c-0.582-0.001-1.368-0.001-1.95,0 c-0.491,0.883-0.782,1.4-1.278,2.28C8.572,7.347,7.755,5.337,6.951,3.399c-0.586,1.29-1.338,3.017-1.923,4.307 c-1.235,0-2.38-0.002-3.615,0"/>
+</svg>
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/images/loading.svg]: New line required.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 1 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 2 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 3 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 4 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 5 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 6 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 7 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 8 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 9 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js (hunk 10 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/noty/noty-wso2.js (hunk 11 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/noty/noty-wso2.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/images/loading.svg (hunk 12 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/images/loading.svg]: Fixed with #1690

**Pull request merged**